bundles/automation/org.eclipse.smarthome.automation.module.core.test/ESH-INF/automation/Xrules/ItemRuleDefinition.json

```diff
@@ -8,7 +8,7 @@
     ],
     "configuration": {},
     "description": "Sample Rule for items definition.",
-    "on": [  
+    "triggers": [
```

Comment:
No, this is why I am asking. So far no test is failing: https://hudson.eclipse.org/smarthome/job/SmartHomeDistribution-Nightly/lastCompletedBuild/testReport/